scripts/gt_concordance.py

```diff
@@ -0,0 +1,551 @@
+import sys
+import os
+import re
+import subprocess
+import shutil
+
+import click
+import yaml
+import vcf
+import pyexcel_xlsx
+
+from ngi_pipeline.database.classes import CharonSession, CharonError
+from ngi_pipeline.log.loggers import minimal_logger
+from ngi_pipeline.utils.classes import with_ngi_config
+
+log = minimal_logger(os.path.basename(__file__))
+
+@click.group()
+@with_ngi_config
+@click.pass_context
+@click.option('--config', '-c', 'custom_config', help='Path to a config file', type=click.Path())
```

Comment:
we use `@with_ngi_config`, which is a function that takes a config from ngi_pipeline. But  `--config` is an additional option, if we want to specify another config file.